ee/EAP7-1193_JakartaEE8.adoc

```diff
@@ -0,0 +1,48 @@
+= Pass the Jakarta EE 8 certification testsuite
+:author:            Scott Marlow
+:email:             smarlow@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+We will run and pass the Jakarta EE 8 TCK with WildFly and our EAP 7.3 product.  This will be done with https://github.com/eclipse-ee4j/jakartaee-tck.  FYI, the TCK development mailing list is https://www.eclipse.org/lists/jakartaee-tck-dev/maillist.html.  
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1193 Jakarta EE 8 Certification
+* https://github.com/eclipse-ee4j/jakartaee-tck/issues/132 TCK challenge that EAP 7.3.0 needs to pass in Elytron mode, we already are passing in legacy security mode.
+
+=== Dev Contacts
+
+* mailto:{Scott Marlow}[{smarlow@redhat.com}] JPA / WildFly integration lead
+
+=== QE Contacts
+
+mailto:{Martin Simka}[{msimka@redhat.com}]
+mailto:{Marek Kopecky}[{mkopecky@redhat.com}]
+
+=== Affected Projects or Components
+
+Since EAP 7.3.0+ will also be Java EE 8 certified, EAP 7.3.0 should also pass the Jakarta EE 8 platform TCK.
+
+We do plan to make related changes, such as https://issues.jboss.org/browse/WFLY-12300, which is for converting to Jakarta EE 8 specification APIs, as the Java EE 8 specification APIs repos on github are in archived state (meaning no further changes are expected).  This means that bug fixes for these artifacts will come via Jakarta (https://github.com/eclipse-ee4j).
+
+=== Known Blockers
+
+We need to ensure that we are running the Jakarta EE 8 TCKs correctly.  
+
+=== Hard Requirements
+
+EAP 7.3.0 passes the https://github.com/eclipse-ee4j/jakartaee-tck 8.0.1 TCK and file our TCK results on https://github.com/RedHatOfficial/jbosseap-certifications.
```

Comment:
Hard requirements needs to contain information that both legacy and elytron security need pass Jakarta TCK

Reply:
done